src/actions-on-google.ts

```diff
@@ -20,7 +20,7 @@ const grpc = require('grpc')
 const protoFiles = require('google-proto-files')
 import * as path from 'path'
 import { UserRefreshClient } from 'google-auth-library'
-import * as i18n from 'i18n'
+const i18n = require('i18n')
```

Comment:
The code `from 'i18n'` cannot load the i18n library (if we use `@types/i18n` in the package.json, can load it, probably). Instead, I would like to use the `require` function.

Reply:
We're already importing `@types/promise.prototype.finally`, so we could import `@types/i18n` as well.

Reply:
Ok, I will replace the `require('i18n')` to the `from 'i18n'` with the `@types/i18n`.